os/android/iahwc2.cpp

```diff
@@ -682,7 +682,7 @@ HWC2::Error IAHWC2::HwcDisplay::PresentDisplay(int32_t *retire_fence) {
     layers.emplace_back(l.second->GetLayer());
   }
 
-  if (layers.empty() && display_->Type() != DisplayType::kLogical)
+  if (layers.empty() && (display_->Type() != DisplayType::kLogical))
```

Comment:
existing expression is correct and clear enough. no need to submit a dedicated pull request to change it.
but of coz you are welcome to refine the code.